src/Cli/dotnet/commands/dotnet-list/dotnet-list-package/ListPackageReferencesCommandParser.cs

```diff
@@ -61,6 +61,11 @@ internal static class ListPackageReferencesCommandParser
                 ArgumentHelpName = CommonLocalizableStrings.LevelArgumentName
             }.ForwardAsSingle(o => $"--verbosity:{o}");
 
+        public static readonly Option FormatOption = new ForwardedOption<string>("--format", LocalizableStrings.CmdFormatDescription)
```

Comment:
If you define an enum for the allowed format options, you get better help output and input validation for free from System.CommandLine. It will tell users what the allowed values are in the help description.

Reply:
I added `enum` option. Could you please check if I implemented it correctly?